bower.json

```diff
@@ -7,9 +7,13 @@
     "output"
   ],
   "dependencies": {
+    "purescript-freet": "safareli/purescript-freet#patch-1",
```

Comment:
let's wait until https://github.com/purescript-contrib/purescript-freet/pull/18 is merged and remove resolution